app/controllers/users_controller.rb

```diff
@@ -53,7 +53,7 @@ def choose_identity
   
   def update_identity
     new_identity = h(params[:identity].to_s)
-    return redirect_to choose_identity_path unless %w(Student Volunteer Mentor).include?(new_identity)
+    return redirect_to choose_identity_path unless %w(Participant Volunteer Mentor).include?(new_identity)
```

Comment:
I would suggest reverting this back to `Student Volunteer Mentor` and keeping Participants separate from this logic